src/components/Card/useStyles.js

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ breakpoints, typography }) => ({
+  root: {},
+  cardMedia: {
+    // TODO(kilemensi): Confirm if all image will have invisible margins
+    //                  around them. If yes, leave this as is. Otherwise
+    //                  remove all negative margins calculations
```

Comment:
Have we confirmed if all images will have margins?

Reply:
Seems only the Explore images have the extra margins. The rest of the images are fine.

Reply:
@gertie-sheshe I should have tagged you on [this](https://trello.com/c/sznZ9PbN/43-xd-links-for-devs) Images have box-shadows and not margins cc @kilemensi